index.html

```diff
@@ -690,7 +691,17 @@ <h3 class="section-subheading text-muted"></h3>
 
     <script src="js/contributors.js"></script>
     <script src="js/scrolltoTop.js"></script>
-
+    <!-- library for running text in js  -->
+    <script src="https://cdn.jsdelivr.net/npm/typed.js@2.0.12"></script>
+    <!-- added in running text on headline  -->
+    <script>
+        var typed = new Typed('.runningtext', {
+  strings: ["For Everyone"],
```

Comment:
Great Job @Shobhnik13 , more quotes can be added.
I have left some suggestion below
````suggestion
  strings: ["for everyone","is about collaboration not competition","For the Community by the Community"],
````

feel free to suggest anything better.